src/test/java/br/com/sbs/cubatech/CourseTest.java

```diff
@@ -0,0 +1,93 @@
+package br.com.sbs.cubatech;
+
+import br.com.sbs.cubatech.category.Category;
+import br.com.sbs.cubatech.category.SubCategory;
+import br.com.sbs.cubatech.course.Course;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CourseTest {
+
+    private Category programacao;
+    private SubCategory sub1;
```

Comment:
Acho que o nome `sub1` é ruim. Eu colocaria:

```suggestion
    private SubCategory subcategory;
```

Reply:
Alterado

Reply:
Alterado